vcap/vcap/loading/capsule_loading.py

```diff
@@ -288,3 +293,40 @@ def capsule_module_name(data: bytes) -> str:
     """Creates a unique module name for the given capsule bytes"""
     hash_ = hashlib.sha256(data).hexdigest()
     return f"capsule_{hash_}"
+
+
+_TYPE_CALLABLE = Union[type, callable]
+
+
+def _validate_capsule_field(capsule: BaseCapsule,
+                            name: str,
+                            value: Any,
+                            type_: _TYPE_CALLABLE) -> None:
+    if type_ is callable:
+        if not callable(value):
```

Comment:
Nitpick: I'd put these on the same line

Reply:
I actually had it like that originally, but realized the logic didn't quite work out. If I do:

```
if type_ is callable and not callable(value):
    # Raise error
elif not isinstance(value, type_):
    # Raise error
```

And the `type_ == callable` and `value` is a callable object, it will result in a call to `isinstance` with `callable` as the type argument, which fails.

Reply:
Oh you're totally right